lib/routes/lnurl_fetch_invoice_page.dart

```diff
@@ -218,17 +218,19 @@ class LNURLFetchInvoicePageState extends State<LNURLFetchInvoicePage> {
                     },
                     style: theme.FieldTextStyle.textStyle,
                   ),
-                  Padding(
-                    padding: const EdgeInsets.only(top: 8),
-                    child: Text(
-                      texts.lnurl_fetch_invoice_limit(
-                        acc.currency.format(response.minAmount),
-                        acc.currency.format(response.maxAmount),
-                      ),
-                      textAlign: TextAlign.left,
-                      style: theme.FieldTextStyle.labelStyle,
-                    ),
-                  ),
+                  !response.isFixedAmount
+                      ? Padding(
+                          padding: const EdgeInsets.only(top: 8),
+                          child: Text(
+                            texts.lnurl_fetch_invoice_limit(
+                              acc.currency.format(response.minAmount),
+                              acc.currency.format(response.maxAmount),
+                            ),
+                            textAlign: TextAlign.left,
+                            style: theme.FieldTextStyle.labelStyle,
+                          ),
+                        )
+                      : Padding(padding: const EdgeInsets.only(top: 8)),
```

Comment:
Is the padding necessary when the amount is fixed? If it's not necessary, I suggest to add this widget to the widget tree as such:
```suggestion
                    if (!response.isFixedAmount) ...[
                      Padding(
                           padding: const EdgeInsets.only(top: 8),
                           child: Text(
                             texts.lnurl_fetch_invoice_limit(
                               acc.currency.format(response.minAmount),
                               acc.currency.format(response.maxAmount),
                             ),
                             textAlign: TextAlign.left,
                             style: theme.FieldTextStyle.labelStyle,
                           ),
                         )
                    ]
```